app/services/images/optimizer.rb

```diff
@@ -35,12 +35,26 @@ def self.cloudinary(img_src, **kwargs)
 
     def self.imgproxy(img_src, **kwargs)
       options = DEFAULT_IMGPROXY_OPTIONS.merge(kwargs).reject { |_, v| v.blank? }
-
+      Imgproxy.config.endpoint ||= set_imgproxy_endpoint
       Imgproxy.url_for(img_src, options)
     end
 
     def self.imgproxy_enabled?
       Imgproxy.config.key.present? && Imgproxy.config.salt.present?
     end
+
+    def self.set_imgproxy_endpoint
+      if Rails.env.production?
```

Comment:
We no longer need to check for `ApplicationConfig["APP_DOMAIN"] && ApplicationConfig["APP_PROTOCOL"]` because at this point the App is completely booted with the proper domain and protocol set. I've previously added that to fix container building issue.